docs/schemes/hes.rst

```diff
@@ -17,5 +17,5 @@ Homomorphic Encryption Standard
    >>> HESv111024128ternary
    LWEParameters(n=1024, q=134217728, Xs=D(σ=0.82), Xe=D(σ=3.00), m=1024, tag='HESv11ternary')
    >>> primal_hybrid(HESv111024128ternary)
-   rop: ≈2^130.8, red: ≈2^130.5, svp: ≈2^128.4, β: 338, η: 369, ζ: 0, |S|: 1, d: 2049, prob: 1, ↻: 1, tag: hybrid
+   rop: ≈2^183.4, red: ≈2^182.6, svp: ≈2^182.3, β: 345, η: 2, ζ: 139, |S|: ≈2^220.3, ...
```

Comment:
So hybrid now enforces Babai? Shouldn't it allow large \eta, too?

Reply:
We decided to set `babai = True` to be the default purely because `prob.mitm_babai_probability()` only covers the case of `η = 2`. In the case of `η > 2`, this probability is yet-to-be analysed and so we've (conservatively) set this probability to be `1`. We thought about maybe setting `babai = False` and allowing the computation of `prob.mitm_babai_probability()` only when `η = 2`, but this might be hiding a few too many assumptions under-the-hood, or could potentially lead to weird optimizations.

Definitely open to alternatives!